manubot/cite/handlers.py

```diff
@@ -25,6 +25,25 @@
     "manubot.cite.wikidata.Handler_Wikidata",
 ]
 
+_infer_prefix_patterns = {
```

Comment:
Is there a reason this is a set instead of a list that specifies the precedence of patterns?  Currently all of the patterns match unique strings, right?  Will that always be true in the future?

Reply:
mistake. fixed in https://github.com/manubot/manubot/pull/277/commits/fcf9a9421ef8d5a9cbc9dabdf95f07af2153ce60